bbq/vm/test/vm_test.go

```diff
@@ -5311,5 +5210,115 @@ func TestCompileForce(t *testing.T) {
 		require.NoError(t, err)
 		assert.Equal(t, vm.NewIntValue(42), actual)
 	})
+}
+
+func TestReturnStatements(t *testing.T) {
```

Comment:
Could you please add (a) test case(s) using the `result` variable in the post condition? (maybe they exist already in another test function)

Reply:
There is a bunch of tests under the post-conditions tests: https://github.com/onflow/cadence/blob/bde5c821fbef69e6ccb00cf80f49850b5d6298bd/bbq/vm/test/vm_test.go#L3631-L3734

Reply:
Nice!